### ISSUE TITLE
Attempting to load a WDL containing imports fails the wrong way if you don't supply any import resolvers

### ISSUE THREAD

**Issue by katevoss** [broadinstitute/cromwell#2701]
@helgridly commented on [Thu Apr 13 2017](https://github.com/broadinstitute/wdl4s/issues/103)

I'm loading a WDL in wdl4s using `WdlNamespaceWithWorkflow.load(my_wdl, Seq())` -- i.e. passing no import resolvers. If the WDL contains imports, I'd expect it to fail and complain that it can't resolve the imports because I haven't specified any resolvers. (In the current code, that'd be [here](https://github.com/broadinstitute/wdl4s/blob/develop/src/main/scala/wdl4s/WdlNamespace.scala#L198).)

However, what actually happens is that wdl4s tries to turn the import into a `File` object [here](https://github.com/broadinstitute/wdl4s/blob/develop/src/main/scala/wdl4s/Import.scala#L18). If that's not a valid kind of file path (perhaps because of a custom URI scheme, or because you're running Windows and colons aren't allowed in filenames), wdl4s blows up with some java-native exception. (In the Windows case, that's `java.nio.file.InvalidPathException`.)

TLDR: wdl4s should throw a useful error if your WDL contains imports but you haven't specified resolvers. It probably shouldn't attempt to load the imports outside the context of a resolver, either.

---

@helgridly commented on [Thu Apr 13 2017](https://github.com/broadinstitute/wdl4s/issues/103#issuecomment-293936559)

I discussed this with @Horneth and he's provided me with a workaround: I can check for the existence of imports in my code by loading the AST directly using something like `Option(ast).map(_.getAttribute("imports")).toSeq` (lifted from [here](https://github.com/broadinstitute/wdl4s/blob/develop/src/main/scala/wdl4s/WdlNamespace.scala#L185)). So consider this a non-urgent enhancement rather than a cloud of fire.

**Comment by katevoss** [comment]
@helgridly is this still a non-urgent enhancement request?

**Comment by helgridly** [comment]
Yeah, this was from way back when we on FC thought we'd be parsing the WDL and doing docker-hash switcheroos ourselves. But you ended up doing it! So my use case for needing this has gone away.
